src/Framework/Assert/Functions.php

```diff
@@ -9,6 +9,7 @@
  */
 use PHPUnit\Framework\AssertionFailedError;
 use PHPUnit\Framework\Constraint\Constraint;
+use PHPUnit\Framework\Constraint\Count;
```

Comment:
For reference, see [`PHPUnit\Framework\Assert\Functions.php`](https://github.com/sebastianbergmann/phpunit/blob/6.0.11/src/Framework/Assert/Functions.php#L1711).